sklearn/cluster/hierarchical.py

```diff
@@ -642,9 +642,7 @@ class AgglomerativeClustering(BaseEstimator, ClusterMixin):
           all observations of the two sets.
 
     pooling_func : callable, default=np.mean
-        This combines the values of agglomerated features into a single
-        value, and should accept an array of shape [M, N] and the keyword
-        argument ``axis=1``, and reduce it to an array of size [M].
+        ignored, Deprecate to be removed in 0.22.
```

Comment:
*deprecated

Reply:
Look at other places we use the "deprecated" sphinx directive. `git grep '\.\. deprecated'` will show you plenty of examples.